components/table/test/table.vdiff.js

```diff
@@ -424,181 +425,223 @@ describe('table', () => {
 				});
 			});
 
-			describe('sticky', () => {
-				const inputRow = html`
-					<tr class="down">
-						<td>Cell 2-A</td>
-						<td><d2l-input-text label="label" label-hidden value="Cell 2-B" input-width="100px"></d2l-input-text></td>
-						<td>Cell 2-C</td>
-					</tr>
-				`;
-				[
-					{
-						name: 'one-row-thead',
-						template: html`
-								<thead>${createHeaderRow({ trClass: 'top' })}</thead>
+			// only test the default type to minimize permutations (hasStickyHeadersScrollWrapper isn't expected to have a different impact on the light vs default type)
+			(type === 'default' ? [true, false] : [false]).forEach((hasStickyHeadersScrollWrapper) => {
+				describe(`sticky${hasStickyHeadersScrollWrapper ? '-scroll-wrapper' : ''}`, () => {
+
+					// Currently some of the scenarios with 'hasStickyHeadersScrollWrapper' setting enabled have unexpected outputs
+					// This is used to call those out by wrapping a warning grouping around them
+					function itWithWarnings(testName, testFunc) {
```

Comment:
I'm aiming to call out the specific tests that I believe currently have unexpected outputs, as mentioned in an earlier review [here](https://github.com/BrightspaceUI/core/pull/4481#discussion_r1516912332). 

This might be a touch fragile, but I liked how clearly it calls them out in the report  🤷   I welcome any and all alternative suggestions  :)